compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1640,15 +1640,14 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
         let opaque_ty_def_id = self.resolver.local_def_id(opaque_ty_node_id);
 
         let mut collected_lifetimes = FxHashMap::default();
-        let mut universal_paths = Vec::new();
         self.with_hir_id_owner(opaque_ty_node_id, |lctx| {
             lctx.with_fresh_generics_def_id_map(|lctx| {
                 let type_const_params =
                     lctx.remap_type_and_const_params(opaque_ty_def_id, &ast_generics.params);
 
                 debug!(?type_const_params);
 
-                let (universal_params, universal_predicates, universal_paths_inner): (
+                let (universal_params, universal_predicates, universal_paths): (
```

Comment:
this is actually unused now, or at least not useful outside of the context of the RPIT itself

```suggestion
                let (universal_params, universal_predicates, _): (
```